.env.example

```diff
@@ -4,6 +4,7 @@ SENDGRID_USERNAME=
 SENDGRID_PASSWORD=
 EMAIL_FROM_ADDRESS=
 
+AWS_REGION=us-east-2
```

Comment:
Would it be safer to remove "us-east-2" from the .env.example? As it does need to be filled out to match whatever the developer chooses to setup, otherwise they may think that it has to be us-east-2 😊

Reply:
Yes definitely - that will ensure people know that's a unique value to them and not some default setting they don't need to change if they don't want to.